### PR TITLE
[macOS] Rework Xamarin documentation

### PR DESCRIPTION
**Pull request by nikita-bykov**
# Description
We've improved the documentation for Xamarin by adding the table containing Xamarin bundles and associated symlinks.

image

#### Related issue:
[Add information about default Xamarin / Mono to MacOS image documentation #1929](https://github.com/actions/virtual-environments-internal/issues/1929)
## Check list
- [x] Related issue / work item is attached
- [x] Documentation is updated (if applicable)
- [x] Changes are tested and related VM images are successfully generated

### CODE REVIEW

**Comment by al-cheb** [comment]
Looks like the table contains inaccurate information. For e.g:
```
                "symlink": "6_4_1",
                "mono": "6.4",
                "ios": "13.4",
                "mac": "6.4",
                "android": "10.0"
```

**Comment by nikita-bykov** [comment]
> Looks like the table contains inaccurate information. For e.g:
> 
> ```
>                 "symlink": "6_4_1",
>                 "mono": "6.4",
>                 "ios": "13.4",
>                 "mac": "6.4",
>                 "android": "10.0"
> ```

This table was for an example, the data in it was entered manually and randomly. I've updated it

File: images/macos/software-report/SoftwareReport.Xamarin.psm1 [hunk 1]
```diff
@@ -13,66 +10,26 @@ function Get-NUnitVersion {
...
-        $versionFilePath = Join-Path $_.FullName "Version"
-        if (Test-Path $versionFilePath) {
-            return Get-Content -Raw $versionFilePath
+function Build-XamarinTable {
+    $xamarinBundles = Get-ToolsetValue "xamarin.bundles"
+    $defaultSymlink = Get-ToolsetValue "xamarin.bundle-default"
+    if ($defaultSymlink -eq "latest") {
+        $defaultSymlink = $xamarinBundles[0].symlink
+    }
+    $sortRules = @{
```
Comment (maxim-lobanov) [inline images/macos/software-report/SoftwareReport.Xamarin.psm1:19]: Why do we need sorting here? Can we just print bundle in the order how they are located in toolset.json?
Sorting by symlink as a string can be unreliable

**Review by maxim-lobanov — APPROVED** [review]

File: images/macos/software-report/SoftwareReport.Xamarin.psm1 (hunk 1 above)
Comment (nikita-bykov) [reply images/macos/software-report/SoftwareReport.Xamarin.psm1]: Removed this sort

**Review by AlenaSviridenko — APPROVED** [review]

**Review by vsafonkin — APPROVED** [review]

**Pull request merged**